core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -59,15 +59,21 @@
   /**
    * {@link Attributes.Key} for the remote address of server call attributes
    * {@link ServerCall#attributes()}
+   *
+   * @deprecated use {@link io.grpc.Grpc#TRANSPORT_ATTR_REMOTE_ADDR} instead
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1710")
+  @Deprecated
   public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
           Attributes.Key.of("remote-addr");
```

Comment:
This needs to be `Grpc.TRANSPORT_ATTR_REMOTE_ADDR`, otherwise it won't match. Similar below.